CMakeLists.txt

```diff
@@ -427,8 +428,10 @@ IF( NOT BUILD_DOCUMENTATION_ONLY )
   SET( CaPTk_CLIs_Folder "CaPTk_CLIApps")
   SET( CaPTk_Libs_Folder "CaPTk_Libs")
 
-  ADD_SUBDIRECTORY(${PROJECT_SOURCE_DIR}/src)
-  SET(INDIVIDUAL_APPLICATIONS_DIR ${PROJECT_SOURCE_DIR}/src/applications/individualApps)
+  IF (NOT TARGET plugin)
```

Comment:
Where is this target coming from? I don't see any references to it.

Reply:
hmm I don't remember making that change. Let me remove that

Reply:
nvm you just merged my fork

Reply:
Can't be; it came in after I pulled from your fork: https://github.com/sarthakpati/CaPTk/commit/818c0b98986fdb75aa67f874186a45bf7f494d73

Reply:
Origin of this commit: https://github.com/PhucNgo1711/CaPTk/commit/52dc7430473e6f0c15b495313257fa6d5ed8e511#diff-af3b638bc2a3e6c650974192a53c7291

Reply:
And fixed!